<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(16, 128, 64);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Dice1">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>570</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/Images/Alea_1.png);
color: rgb(128, 0, 2);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Dice2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>570</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/Images/Alea_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Dice3">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>570</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/Images/Alea_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Dice4">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>710</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/Images/Alea_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Dice5">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>710</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Images/Images/Alea_1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Roll">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>320</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(128, 0, 2);
font: 24pt &quot;.SF NS Text&quot;;</string>
    </property>
    <property name="text">
     <string>Roll The Dice</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_scoreboard">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>371</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_EndTurn">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>320</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(128, 0, 2);
font: 24pt &quot;.SF NS Text&quot;;</string>
    </property>
    <property name="text">
     <string>End Turn</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 56pt &quot;.SF NS Text&quot;;
</string>
    </property>
    <property name="text">
     <string>YATZY</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_playername">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;.SF NS Text&quot;;
color: rgb(128, 0, 2);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;.SF NS Text&quot;;</string>
    </property>
    <property name="text">
     <string>Throws Left:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_throws">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>180</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;.SF NS Text&quot;;
color: rgb(128, 0, 2);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1239</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
